### PR TITLE
Multi arch test

### PR DESCRIPTION
**Pull request by adityajn105**

### CODE REVIEW

File: opentutor_classifier/tests/test_train_classifier.py [hunk 1]
```diff
@@ -172,6 +173,49 @@ def test_train_and_predict(
 @pytest.mark.parametrize(
     "lesson,arch,evaluate_inputs,expected_training_result,expected_evaluate_results",
     [
+        (
```
Comment (beatthat) [inline opentutor_classifier/tests/test_train_classifier.py:176]: let's not add any additional tests with the expected results hard coded python, instead create a [test.csv](https://github.com/opentutor/opentutor-classifier/blob/multi_arch_test/opentutor_classifier/tests/fixtures/data/question1/test.csv) for your example and then add it to [this pytest test](https://github.com/opentutor/opentutor-classifier/blob/ed74708b89eaf553fd4a1bf0d5c1b00e306ced75/opentutor_classifier/tests/test_train_classifier.py#L132)

**Review by beatthat — CHANGES_REQUESTED** [review]
looks good! things we need to fix:

 - [ ] change any tests added for this PR to use the test.csv-based test (see inline)

File: opentutor_classifier/tests/test_train_classifier.py (hunk 1 above)
Comment (adityajn105) [reply opentutor_classifier/tests/test_train_classifier.py]: I have added test cases as CSV files. Please check it out.

**Review by beatthat — APPROVED** [review]
looks great! 

I need to do something about separating trained model files by architecture so we can keep multiple archs for the same model (especially default), but I can handle that back in the base PR

**Pull request merged**